policy/modules/kernel/files.if

```diff
@@ -3848,14 +3848,31 @@ interface(`files_dontaudit_read_etc_runtime_files',`
 ##	</summary>
 ## </param>
 #
-interface(`files_dontaudit_execuite_etc_runtime_files',`
+interface(`files_dontaudit_execute_etc_runtime_files',`
 	gen_require(`
 		type etc_runtime_t;
 	')
 
 	dontaudit $1 etc_runtime_t:file execute;
 ')
 
+########################################
+## <summary>
+##	Do not audit attempts to execute files
+##	in /etc that are dynamically
+##	created on boot, such as mtab.
+## </summary>
+## <param name="domain">
+##	<summary>
+##	Domain to not audit.
+##	</summary>
+## </param>
+#
+interface(`files_dontaudit_execuite_etc_runtime_files',`
```

Comment:
It should be "`exec`" instead of "`execute`".

Reply:
changed